crates/tests/ntstatus/tests/ntstatus.rs

```diff
@@ -23,7 +23,8 @@ fn test() -> Result<()> {
 
         let mut random = GUID::zeroed();
 
-        BCryptGenRandom(provider, &mut random as *mut _ as _, core::mem::size_of::<GUID>() as _, 0)?;
+        // TODO: workaround for https://github.com/microsoft/windows-rs/issues/1938
+        BCryptGenRandom(provider, std::mem::transmute(&mut random), core::mem::size_of::<GUID>() as _, 0)?;
```

Comment:
What is `&mut random` being transmuted into?

Reply:
```rust
pub unsafe fn BCryptGenRandom<'a, P0>(halgorithm: P0, pbbuffer: &mut u8, cbbuffer: u32, dwflags: u32) -> ::windows::core::Result<()>
```

But #1938 should turn that into a slice - still require some kind of explicit cast tho.